src/Item_Project.php

```diff
@@ -265,16 +267,97 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
 
     public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
 
         switch ($item->getType()) {
             case 'Project':
                 self::showForProject($item);
                 break;
 
             default:
-               // Not defined and used now
-               // Project::showListForItem($item);
+                if (in_array($item->getType(), $CFG_GLPI["project_asset_types"])) {
```

Comment:
You should update `getTabNameForItem` to add this condition.

Reply:
```
default:
                   // Not used now
                    if (
                        Session::haveRight("project", Project::READALL)
                        && ($item instanceof CommonDBTM)
                    ) {
                        if ($_SESSION['glpishow_count_on_tabs']) {
                              // Direct one
                              $nb = self::countForItem($item);

                              // Linked items
                              $linkeditems = $item->getLinkedItems();

                            if (count($linkeditems)) {
                                foreach ($linkeditems as $type => $tab) {
                                    $typeitem = new $type();
                                    foreach ($tab as $ID) {
                                        $typeitem->getFromDB($ID);
                                        $nb += self::countForItem($typeitem);
                                    }
                                }
                            }
                        }
                        return self::createTabEntry(Project::getTypeName(Session::getPluralNumber()), $nb, $item::getType());
                    }
```
Today's behaviour is already managing the situation

Reply:
I do not see any checks based on `$CFG_GLPI["project_asset_types"]`.
For instance, if you add `->addStandardTab('Item_Project', $ong, $options)` on a random class, it should not return the tab entry if the corresponding class is not declared in `$CFG_GLPI["project_asset_types"]`.

Reply:
My bad, you added this condition later.

Reply:
And you removed it in a subsequent commit. Why ?

Reply:
I added it, then the lint stopped and I realized that it still worked without it. I'm going to put it back in and make sure it passes the lints.